rental.rb

```diff
@@ -0,0 +1,17 @@
+require 'time'
+
+class Rental
+  attr_accessor :person, :book, :date
+
+  def initialize(person, book)
+    @date = DateTime.now.strftime('%Y/%m/%d %H:%M')
+
+    @book.rentals << self
+
+    @person = person
+
+    @book.rentals << self
+
+    @book = book
+  end
```

Comment:
**OPTIONAL** - You have a wrong association here, a `rental` class should belong to a `Person` and a `Book`, you should push an instance of the `rental` to the `person` class also, please fix this 🙏